CMakeLists.txt

```diff
@@ -3,24 +3,24 @@ IF(COMMAND CMAKE_POLICY)
   CMAKE_POLICY(SET CMP0003 NEW)
 ENDIF(COMMAND CMAKE_POLICY)
 
-SET(kit PrincipalComponentsAnalysis)
-PROJECT(${kit})
+SET(PCA PrincipalComponentsAnalysis)
+PROJECT(${PCA})
 
-IF (NOT ITK_SOURCE_DIR)
+IF(NOT ITK_SOURCE_DIR)
   FIND_PACKAGE(ITK REQUIRED)
   LIST(APPEND CMAKE_MODULE_PATH ${ITK_CMAKE_DIR})
   INCLUDE(ITKModuleExternal)
 ELSE()
   itk_module_impl()
 ENDIF()
 
-OPTION(${kit}_BUILD_EXAMPLES "Build the examples" OFF)
-IF(${kit}_BUILD_EXAMPLES)
-  ADD_SUBDIRECTORY( Examples )
+OPTION(Module_${PCA}_BUILD_EXAMPLES "Build the examples" OFF)
+if(BUILD_EXAMPLES AND Module_${PCA}_BUILD_EXAMPLES)
```

Comment:
I think it might be confusing for the developer who tries to build the module examples as they may not understand why the examples are not building even though Module_${PCA}_BUILD_EXAMPLES might be ON.
One solution would be to make the CMake variable Module_${PCA}_BUILD_EXAMPLES dependent of BUILD_EXAMPLES, to hide this option if BUILD_EXAMPLES is OFF with CMAKE_DEPENDENT_OPTION .